awscli/clidriver.py

```diff
@@ -363,8 +363,10 @@ def _get_command_table(self):
 
     def _get_service_model(self):
         if self._service_model is None:
+            api_version = self.session.get_config_variable('api_versions').get(
```

Comment:
The output of `get_config_variable` is `None` if they don't have versions set, so you'll need to check for that.